recipes/opencv/4.x/conanfile.py

```diff
@@ -184,29 +184,30 @@ def requirements(self):
         if self.options.get_safe("with_jpeg2000") == "jasper":
             self.requires("jasper/2.0.33")
         elif self.options.get_safe("with_jpeg2000") == "openjpeg":
-            self.requires("openjpeg/2.4.0")
+            self.requires("openjpeg/2.5.0")
         if self.options.with_png:
             self.requires("libpng/1.6.37")
         if self.options.with_openexr:
-            self.requires("openexr/2.5.7")
+            self.requires("openexr/3.1.5")
         if self.options.get_safe("with_tiff"):
             self.requires("libtiff/4.3.0")
         if self.options.with_eigen:
-            self.requires("eigen/3.3.9")
+            self.requires("eigen/3.4.0")
         if self.options.get_safe("with_ffmpeg"):
-            self.requires("ffmpeg/4.4")
+            self.requires("ffmpeg/5.0")
+            self.requires("libiconv/1.17", override=True)
         if self.options.parallel == "tbb":
-            self.requires("onetbb/2020.3")
+            self.requires("onetbb/2021.3.0")
```

Comment:
is it compatible?

Reply:
I was able to compile it locally. I can't say more